turbo.json

```diff
@@ -1,12 +1,16 @@
 {
   "$schema": "https://turbo.build/schema.json",
   "pipeline": {
-    "test": { "cache": false },
-    "test:unit": {},
-    "test:integration": {},
-    "test:browser": {},
-    "lint": {},
-    "lint:fix": {},
-    "build": {}
+   "test": {"cache": false},
+   "prepublishOnly": {"cache": false},
+   "test:unit": {"cache": false},
+   "test:integration": {"cache": false},
+   "test:browser": {"cache": false},
+   "lint": {"cache": false},
+   "lint:fix": {"cache": false},
+   "build": {"cache": false , "dependsOn": ["@asyncapi/parser#build"]},
+   "@asyncapi/parser#build":{"cache": false},  
+   "multi-parser:build":{"cache":false, "dependsOn": ["@asyncapi/parser#build"]}
```

Comment:
Stupid question: shouldnt this be named `@asyncapi/multi-parser` as well?

Reply:
no, the first @asyncapi/parser#build is referring to the name of the package. but the second multiparser is referring to the command in the script.